bin/run.sh

```diff
@@ -25,13 +25,8 @@ slug="$1"
 snake_slug=${slug//-/_}
 input_dir="${2%/}"
 output_dir="${3%/}"
-if [[ -f "${input_dir}/spec/${snake_slug}_spec.cr" ]]
-then
-    spec_file="${input_dir}/spec/${snake_slug}_spec.cr"
-else
-    spec_files=( ${input_dir}/spec/*_spec.cr )
-    spec_file="${spec_files[0]}"
-fi
+config=$(jq -r '.files.test[0]' ${input_dir}/.meta/config.json)
+spec_file="${input_dir}/${config}"
```

Comment:
Could you check if this works?

```suggestion
spec_file="${input_dir}/$(jq -r '.files.test[0]' ${input_dir}/.meta/config.json)"
```

If not, you could maybe rename the variable a bit:

```suggestion
relative_spec_file=$(jq -r '.files.test[0]' ${input_dir}/.meta/config.json)
spec_file="${input_dir}/${relative_spec_file}"
```